distribution/docs/src/main/resources/_contents/_running/ingesting-contents.adoc

```diff
@@ -30,7 +23,7 @@ For XML data, run this command:
 ingest -t xml examples/metacards/xml
 ----
 
-====== Directory Monitor
+===== Directory Monitor
 
 The ${ddf-catalog} application contains a Directory Monitor feature that allows files placed in a single directory to be monitored and ingested automatically.
 For more information about configuring a directory to be monitored, consult Directory Monitor.
```

Comment:
Did we start new terminology on linking or referencing another section? Not sure I remember `consult`